pydocstringformatter/run.py

```diff
@@ -6,22 +6,27 @@
 import sys
 import tokenize
 from pathlib import Path
-from typing import List, Union
+from typing import List, Optional, Union
 
-from pydocstringformatter import __version__, formatting, utils
+from pydocstringformatter import __version__, utils
+from pydocstringformatter.formatting import FORMATTERS, Formatter
```

Comment:
Personally I like the `formatting` import as it helps show the namespace of stuff, but I'm not too opinionated about this. If you prefer this I'm fine with it!

Reply:
Done

Reply:
Thanks 😄